src/procstat.c

```diff
@@ -872,11 +890,27 @@ static ssize_t reset_u64_series(void *object, uint64_t arg, char *buffer, size_t
 	return 1;
 }
 
+static ssize_t reset_interval_u64_series(void *object, uint64_t arg, char *buffer, size_t length)
+{
+	struct procstat_series *series_stat = object;
+	struct procstat_series_u64 *series = series_stat->private;
+	uint32_t control;
+
+	control = strtoul(buffer, NULL, length);
+	if (control < 0)
+		return EINVAL;
+
+	__atomic_store_n(&series->reset_interval, control, __ATOMIC_RELAXED);
+	return 1;
```

Comment:
ummm... shouldn't you return 0 for success?

Reply:
@adir this this is the size to return so I think it should be greater than 0 to show success and he has copied it from another function that does reset.

Reply:
I used it similarly to reset_u64_series(). Should it return 'length'?

Reply:
@alexs-lb if this function should return positive length, then returning `EINVAL` in case of error is wrong (assuming `EINVAL` is positive value).
so, we either need to return `-EINVAL` in case of failure or change the return value in case of success (i guess the first option is more correct).

Reply:
@adir-lb As far as I understand the code according to function fuse_write(), returning 1 is the only value that indicates success.